src/main/java/com/continuum/nova/chunks/IGeometryFilter.java

```diff
@@ -142,7 +138,7 @@ public boolean matches(IBlockState blockState) {
 
         @Override
         public String toString() {
-            if(shouldBeTransparent) {
```

Comment:
Decide then if you want spaces in `if statement` or not. In one class you adding spaces, but in another you removing them.

Reply:
I added spaces before because it looks better than without it (IMHO). But i removed it later because @DethRaid said "Don't change my spacing".